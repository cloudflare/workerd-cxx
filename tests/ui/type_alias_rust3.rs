struct Type;

#[cxx::bridge]
mod ffi {
    extern "Rust" {
        // Not allowed - the target is not a path.
        type Source = &crate::Type;
    }
}

fn main() {}
