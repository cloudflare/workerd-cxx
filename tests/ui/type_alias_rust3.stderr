error: unsupported `extern "Rust"` alias target
 --> tests/ui/type_alias_rust3.rs:7:23
  |
7 |         type Source = &crate::Type;
  |                       ^^^^^^^^^^^^
